aea/aea_builder.py

```diff
@@ -1246,6 +1243,70 @@ def set_from_configuration(
                 skip_consistency_check=skip_consistency_check,
```

Comment:
a very quick workaround would have been to put this flag to `False` for skill components. But that would have broken other consistencies.

Another idea is to first load abstract skills, then concrete skills. But it lacks of generality (non-abstract skills can depend on non-abstract skills), and simply doesn't work with our model.

I couldn't come up with simpler approaches than this one.